webapp/src/modules/auction/sagas.js

```diff
@@ -71,3 +90,49 @@ function* handleBidRequest(action) {
     yield put(bidOnParcelsFailure(error.message))
   }
 }
+
+function* handleFetchAuctionRateRequest(action) {
+  const { token } = action
+  const selectedToken = yield select(state => getSelectedToken(state))
```

Comment:
Just to be consistent
```suggestion
  const selectedToken = yield select(getSelectedToken)
```

Reply:
✅ done